benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(SUBSYS_NAME benchmarks)
+set(SUBSYS_DESC "Point cloud library benchmarks")
+set(SUBSYS_DEPS common features search kdtree io)
+set(DEFAULT OFF)
+set(build TRUE)
+set(REASON "Disabled by default")
+PCL_SUBSYS_OPTION(build "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
+PCL_SUBSYS_DEPEND(build "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS})
+if(NOT build)
+  return()
+endif()
+
+find_package(benchmark REQUIRED)
+add_executable(benchmark_features features.cpp)
+target_link_libraries(benchmark_features benchmark::benchmark pcl_io pcl_search pcl_features)
+set_target_properties(benchmark_features PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+#add_test(NAME benchmark_features COMMAND benchmark_features ${PCL_SOURCE_DIR}/test/table_scene_mug_stereo_textured.pcd CONFIGURATIONS benchmarks)
+add_custom_target(run_benchmark_features benchmark_features ${PCL_SOURCE_DIR}/test/table_scene_mug_stereo_textured.pcd)
+add_custom_target(run_benchmarks)
+add_dependencies(run_benchmarks run_benchmark_features)
```

Comment:
Please check the PCL_ADD_TESTS in cmake/pcl_targets.cmake

Reply:
Yes, I saw that macro. Do you suggest I use it for the benchmarks? But then the benchmarks would always run together with the tests, which we don't want. My goal here is to provide a target that can be run like `make run_benchmarks`.

Reply:
That makes sense. I was talking more about creating a macro out of these statements.

Reply:
Oh, then yes, definitely :+1: